### PR TITLE
[INLONG-8276][Manager] InlongGroup support multiple tenant

### LINKED ISSUE #8276: [Feature][Manager] InlongGroup support multi-tenancy
### Description

InlongGroup support multi-tenancy
Parent #7914 #8275

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8276 
- Parent #7914 
### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java [hunk 1]
```diff
@@ -18,82 +18,141 @@
...
-            // else, append the tenant condition
-            whereSql.append(" and ( ").append(where).append(" )");
-            Expression expression = CCJSqlParserUtil.parseCondExpression(whereSql.toString());
-            plain.setWhere(expression);
+    private boolean isPrimitiveOrWrapper(Object obj) {
+        try {
+            Class<?> clazz = obj.getClass();
+            return (obj instanceof String)
+                    || clazz.isPrimitive()
+                    || ((Class<?>) clazz.getField("TYPE").get(null)).isPrimitive();
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java:148]: Please add comment.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptorConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+import java.util.List;
+
+@Configuration
+public class MultiTenantInterceptorConfiguration {
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptorConfiguration.java:30]: Please add more javadoc.

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml [hunk 3]
```diff
@@ -25,6 +25,7 @@
         <result column="inlong_group_id" jdbcType="VARCHAR" property="inlongGroupId"/>
         <result column="name" jdbcType="VARCHAR" property="name"/>
         <result column="description" jdbcType="VARCHAR" property="description"/>
+        <result column="tenant" jdbcType="VARCHAR" property="mqType"/>
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml:28]: `<result column="tenant" jdbcType="VARCHAR" property="tenant"/>`

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java [hunk 4]
```diff
@@ -18,82 +18,141 @@
...
-/**
- * Interceptor for multi-tenant.
- */
+@Slf4j
 @Intercepts({
-        @Signature(type = StatementHandler.class, method = "prepare", args = {Connection.class, Integer.class})
+        @Signature(type = ParameterHandler.class, method = "setParameters", args = PreparedStatement.class),
+        @Signature(type = Executor.class, method = "query", args = {MappedStatement.class, Object.class,
+                RowBounds.class, ResultHandler.class, CacheKey.class, BoundSql.class}),
+        @Signature(type = Executor.class, method = "query", args = {MappedStatement.class, Object.class,
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java:59]: Redundant or not？

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java:158]: fixed

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptorConfiguration.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptorConfiguration.java:36]: fixed, thx

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml (hunk 3 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml]: fixed, thx

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**